composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require": {
         "neos/neos-development-collection": "2.0.x-dev",
-        "neos/flow-development-collection": "@dev",
+        "neos/flow-development-collection": "3.0.x-dev",
```

Comment:
this is unnecessary since that dependency already comes from the `neos-development-collection` https://github.com/neos/neos-development-collection/blob/2.0/composer.json#L7